portfolio/src/main/webapp/js/background.js

```diff
@@ -222,17 +241,24 @@ class Ball {
    * @private
    */
   _bounceBounds (tank) {
-    if (this.position.x - tank.ballSize / 2 < 0) {
-      this.velocity.x = -this.velocity.x;
+    const velocityDampingMultiplier = (1 - BALL_BOUNCE_VELOCITY_DAMPING);
```

Comment:
nit: should we just call `BALL_BOUNCE_VELOCITY_DAMPING = 1 - .02` above? Since this is the only place where it's used.

Reply:
Okay. That's the first time I've seen a constant defined like that. If that is considering acceptable style than that does accomplish the same task more cleanly. Changing.